doc/admin-guide/plugins/tcpinfo.en.rst

```diff
@@ -103,6 +103,31 @@ The following options may be specified in :file:`plugin.config`:
   not required to be in the configuration file.  To achieve a log rate
   of 1% you would set this value to 10.
 
+--rolling-enabled=VALUE
+  This logfile option allows you to set logfile rolling behaviour of
+  the ``tcp-metrics.log`` without making any changes to the global
```

Comment:
The log file for the plugin is not called `tcp-metrics.log`, so it is confusing to name it here. Just refer to it as "the output log file".